collections/DOrville/MS_DOrville_153.xml

```diff
@@ -37,45 +40,132 @@
+                        <note>No commentary on book 5.</note>
+                        <!-- not Trevet or Wheatley (there are some resemblances to Wheatley) -->
+                     </msItem>
+                     
+                  </msItem>
+                  <msItem>
+                     <locus>(fol. 133r–v)</locus>
+                     <note>Added inscriptions and pen-trials etc., including:</note>
+                     <msItem>
+                        <note>(fol. 133r)</note>
+                        <note>‘Ve t<ex>ibi</ex> cu nigre dicebat cacubus olle’; ‘Aristotoles(?) Appetitus divitiarum <sic>cressit</sic>
+                           ad infinitum’; ‘Qui scit frenare linguam sensumque domare fortior est e(?) qui vincit  viribus orbes’; ‘in quibus estis qui(?) estis’; ‘Morbos multos multa fercula fecerunt’ (Seneca, Epistulae morales); ‘Si hec duo pronomina meum et tuum tollentur(?) <gap reason="editorial"/>’</note></msItem>
+                     <msItem>
+                        <locus>(fol. 133v)</locus>
+                        <note>Pen-trials etc., including the names ‘Johan’, ‘Willem’(?) and ‘Hoest’, and grammatical notes.</note>
+                     </msItem>
                   </msItem>
                </msContents>
                <physDesc>
+                  <p><secFol>extulisset ipsum</secFol></p>
                   <objectDesc form="codex">
                      <supportDesc material="chart">
-                        <support>paper</support>
+                        <support>Paper folded in quarto. The 'original' volume has three watermarks (a) fols. 1-88, letter T(Tau)/St Anthony's cross (cf. Piccard Kreuz I.47-51); (b) fols. 89-124, two-headed heraldic eagle (cf. Briquet 230, Etain, 1429) (c) fols. 125-132, anchor with cross. The inserted sheets apparently have other watermarks, one of which (fol. 26) is quadruped (?dog/fox; perhaps cf. Piccard Vierfüssler VIII.1540, used in 1450)</support>
+                        <extent> i + ii (medieval paper) + 132 (107 + 25) + i (medieval paper) + i (fols. 7, 9, 11, 13-14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 40, 49, 51, 60, 63, 65, 69, 72, 74, 77 are inserted leaves of various sizes)
+                           <dimensions type="leaf" unit="mm">
+                              <height>215</height>
+                              <width max="145" min="140">140-5</width>
+                           </dimensions></extent>
+                        <foliation>1–132 in 18th(?) century ink; i–iii and 133–134 in 20th-century pencil</foliation>
+                        <collation mainStructures="12">The collation of the 'original' volume is 1(8) (fols. 1-6, 8, 10), 2(12-1)(last cancelled?) (fols. 12, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33), 3(12) (fols. 35-9, 41-7), 4(8) (fols. 48, 50, 52-7), 5(12) (fols. 58-9, 61-2, 64, 66-8, 70-1, 73, 75), 6(12) (fols. 76, 78-88), 7(12) (fols. 89-100), 8(12) (fols. 101-112), 9(12) (fols. 113-124), 10(8)(fols. 125-132)</collation>
```

Comment:
(Does the system do anything with the `mainStructures` attribute, or is that aspirational markup? Wondering whether I need to start including this.)

Reply:
The system doesn't do anything with it currently (just as it doesn't do anything with the attributes on `layout`) but it seems like the sort of information that is worth encoding - when one remembers to do so ...